conda_forge_tick/auto_tick.py

```diff
@@ -921,92 +932,105 @@ def main(args: "CLIArgs") -> None:
-                                pr_json = {
-                                    "state": "closed",
-                                    "head": {"ref": "<this_is_not_a_branch>"},
-                                }
-                            d["PR"] = pr_json
-                            attrs.setdefault("PRed", []).append(d)
-                        attrs.update(
-                            {
-                                "smithy_version": mctx.smithy_version,
-                                "pinning_version": mctx.pinning_version,
-                            },
+                    for base_branch in base_branches:
+                        print("\n", flush=True, end="")
+                        logger.info(
+                            "%s%s IS MIGRATING %s:%s",
+                            migrator.__class__.__name__.upper(),
+                            extra_name,
+                            fctx.package_name,
+                            base_branch,
                         )
-
-                except github3.GitHubError as e:
-                    if e.msg == "Repository was archived so is read-only.":
-                        attrs["archived"] = True
-                    else:
-                        logger.critical(
-                            "GITHUB ERROR ON FEEDSTOCK: %s",
-                            fctx.feedstock_name,
-                        )
-                        if is_github_api_limit_reached(e, mctx.gh):
-                            break
-                except URLError as e:
-                    logger.exception("URLError ERROR")
-                    attrs["bad"] = {
-                        "exception": str(e),
-                        "traceback": str(traceback.format_exc()).split("\n"),
-                        "code": getattr(e, "code"),
-                        "url": getattr(e, "url"),
-                    }
-
-                    pre_key = "pre_pr_migrator_status"
-                    if pre_key not in attrs:
-                        attrs[pre_key] = {}
-                    attrs[pre_key][migrator_name] = sanitize_string(
-                        "bot error: %s" % str(traceback.format_exc()),
-                    )
-                except Exception as e:
-                    logger.exception("NON GITHUB ERROR")
-                    attrs["bad"] = {
-                        "exception": str(e),
-                        "traceback": str(traceback.format_exc()).split("\n"),
-                    }
-
-                    pre_key = "pre_pr_migrator_status"
-                    if pre_key not in attrs:
-                        attrs[pre_key] = {}
-                    attrs[pre_key][migrator_name] = sanitize_string(
-                        "bot error: %s" % str(traceback.format_exc()),
-                    )
-                else:
-                    if migrator_uid:
-                        # On successful PR add to our counter
-                        good_prs += 1
+                        try:
```

Comment:
This seems like a double try block?

Reply:
Right, it is a double try block. I didn't know if we wanted to clean out the tmp dirs after each branch or not. Thoughts?

Reply:
hmm, I don't know. On the one hand it seems safer to clear them out, but I'm not certain what that safety is guarding against.

Reply:
The code is going to refetch the repo from github anyways, so I think maybe we should clean them out so that everything is consistent with the old way of doing things. We can come back later to optimize both the fetch and cleaning if we care. That ok?